crates/bevy_pbr/src/render/light.rs

```diff
@@ -910,7 +910,7 @@ pub fn prepare_lights(
             // we don't use the alpha at all, so no reason to multiply only [0..3]
             color: Vec4::from_slice(&light.color.to_f32_array()) * light.illuminance,
             // direction is negated to be ready for N.L
-            dir_to_light: light.transform.back(),
```

Comment:
This suggests that we should be to storing a direction on this field.

Reply:
> This suggests that we should be to storing a direction on this field.

`Dir3` does not implement `ShaderSize`. To resolve this, it is necessary to import the `bevy_encase_derive` dependency from `bevy_math` and add the `ShaderType` derive to `Dir3`. However, the `ShaderType` derive requires named fields, so the structure of `Dir3` needs to be modified.
If possible, I will submit a new PR to make this change.